docs/admin/access_control/service_accounts.mdx

```diff
@@ -0,0 +1,50 @@
+# Service Accounts
+
+Service accounts are specialized user accounts designed for automation, API integrations, and programmatic access to Sourcegraph, as opposed to using access tokens from regular users. Unlike regular user accounts, service accounts don't require an email address or password and cannot access the Sourcegraph UI. They are specifically intended for automated workflows, CI/CD pipelines, scripts, and other non-human access patterns. Service accounts also don't count towards the user limit and won't be part of any billing cycles.
```

Comment:
nit: maybe some line breaks here for better readability? It's quite a wall of text 😬

Reply:
nit, the two are pretty redundant:

> are specialized user accounts designed for automation, API integrations, and programmatic access to Sourcegraph

> They are specifically intended for automated workflows, CI/CD pipelines, scripts, and other non-human access patterns